atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/CassandraContainer.java

```diff
@@ -56,8 +61,11 @@ public CassandraContainer() {
     private CassandraContainer(String dockerComposeFile, String name) {
         String keyspace = UUID.randomUUID().toString().replace("-", "_");
         this.config = ImmutableCassandraKeyValueServiceConfig.builder()
-                .servers(ImmutableDefaultConfig
-                        .builder().addThrift(forService(name)).build())
+                .servers(ImmutableCqlCapableConfig.builder()
```

Comment:
can these configs and the ones below not be the same? i.e. that and `getConfig` return the same config object. Or better still, can you initialise this config with this stuff?

Reply:
I had some problems with retrieving `Proxy` I would appreciate a double check and we can discuss it in person if it was not exactly what you had in mind.